rmf_fleet_adapter/src/rmf_fleet_adapter/events/GoToPlace.cpp

```diff
@@ -422,60 +487,71 @@ std::optional<rmf_traffic::agv::Plan::Goal> GoToPlace::Active::_choose_goal(
     _description.one_of().size(),
     _context->requester_id().c_str());
 
-  auto lowest_cost = std::numeric_limits<double>::infinity();
-  std::optional<std::size_t> selected_idx;
-  for (std::size_t i = 0; i < _description.one_of().size(); ++i)
+
+  if (_context->_parking_spot_manager_enabled())
   {
-    const auto wp_idx = _description.one_of()[i].waypoint();
-    if (only_same_map)
+    RCLCPP_INFO(_context->node()->get_logger(),
+      "Requesting Chope Node For Time To Progress");
```

Comment:
So if the parking spot manager is enabled this function effectively does nothing and always returns a `std::nullopt`?

Reply:
yep!